charts/s1-agent/templates/hooks/webhookconfiguration.yaml

```diff
@@ -57,4 +57,76 @@ webhooks:
     scope: '*'
 {{- toYaml .Values.agentInjection.selector | nindent 2 }}
   sideEffects: None
+
+---
```

Comment:
This means we will create the hooks if one of the validating/mutating/injection is enabled?
Should we wrap each hook with the relevant if?